blimp-live-widgets/src/components/Clock/index.js

```diff
@@ -46,8 +47,8 @@ export default class Clock extends Component {
   render() {
 
     return (
-      <div>
-        <ClockLayout {...this.state} styles={this.styles} showSmallTicks={this.props.showSmallTicks} />
+      <div className = {styles.wrapper}>
+        <p className = {styles.clock}>{this.state.hour%12}:{this.state.minutes}:{this.state.seconds} { this.state.hour/12 > 0 ? "PM": "AM" }</p>
```

Comment:
This would make noon 0pm instead of 12pm

Reply:
Addressed